test/test_prototype_datasets_builtin.py

```diff
@@ -84,22 +97,53 @@ def test_num_samples(self, dataset_mock, config):
 
         assert len(list(dataset)) == mock_info["num_samples"]
 
+    @pytest.fixture
+    def log_session_streams(self):
+        debug_unclosed_streams = StreamWrapper.debug_unclosed_streams
```

Comment:
@VitalyFedyunin Although this is called "debug", can we depend on this being there and public?